src/gui/application.h

```diff
@@ -72,6 +72,10 @@ public slots:
     // TODO: this should not be public
     void slotownCloudWizardDone(int);
     void slotCrash();
+    /**
+     * Will download a placeholder file, and open the result.
```

Comment:
Suggest to add whether this takes a path to a placeholder or a path to the underlying file.